### ISSUE TITLE
ignore inline comments when creating an alias

### ISSUE THREAD

**Issue by paddykontschak** [jigish/slate#92]
I was trying to document my ~/.slate with some inline comments but it seems like the comments are treated as part of the alias.

```
alias nine 980 # Default
bind h:cmd;ctrl;alt move (1680-${nine})/2;76 ${nine};807
```

This code creates the following error message:

```
**Unable To Create Binding**
Unable to create 'bind h:cmd;ctrl;alt move  (1680-980+14 # Default)/2;76  980+14 # Default;807'
```

![Unable To Create Binding](http://i.imgur.com/3Lfrm.png)

**Comment by jigish** [comment]
Bah. This is totally boneheaded of me. I'll fix this soon.

**Comment by jigish** [comment]
Should be available in the next version. I'll be pushing one out tonight (1.0.4).

**Comment by paddykontschak** [comment]
Just updated to 1.0.4 and tried it again.

**Code:**

```
alias nine 980 # Default
bind h:cmd;ctrl;alt move (1680-${nine})/2;76 ${nine};807
```

**Error message:**

```
Invalid Operation in Chain
Unable to create 'bind h:cmd;ctrl;alt move (1680-980 )/2;76 980 ;807'
```

Looks like the space before the `#` is being added to the alias which results in `980 ;807` instead of the required `980;807`

**Screenshot:**
![Invalid Operation in Chain](http://i.imgur.com/bNrgH.png)

**Comment by jigish** [comment]
Wow. can't believe I overlooked that. fixing now.

**Comment by jigish** [comment]
fixed in 1.0.5

**Comment by paddykontschak** [comment]
Awesome. I'm not getting any errors this time. Thank you for fixing this.
